packages/c/cdrtools/package.yml

```diff
@@ -1,20 +1,21 @@
 name       : cdrtools
-version    : '3.02_9'
-release    : 5
+version    : 3.02a_09
+release    : 6
 source     :
-    - https://jztkft.dl.sourceforge.net/project/cdrtools/alpha/cdrtools-3.02a09.tar.gz : c7e4f732fb299e9b5d836629dadf5512aa5e6a5624ff438ceb1d056f4dcb07c2
+    - https://sourceforge.net/projects/cdrtools/files/alpha/cdrtools-3.02a09.tar.gz/download : c7e4f732fb299e9b5d836629dadf5512aa5e6a5624ff438ceb1d056f4dcb07c2
+homepage    : https://cdrtools.sourceforge.net/private/cdrtools.html
 license    : CDDL-1.0
 component  : library
 summary    : CD Recording Utilities
 description: |
     CD Recording Utilities
 build      : |
     %make LINKMODE=dynamic -j1
-install    : |
+install    : |-
```

Comment:
Be aware that there is a bug in `task bump` wich adds `-` at `install    : |` key.